src/Agent/NewRelic/Agent/Core/Samplers/GCSamplerNetCore.cs

```diff
@@ -110,20 +110,20 @@ public override void Start()
                 }
 
                 _listener = _listener ?? _eventListenerFactory();
+                _listener.StartListening();
             }
             catch (Exception ex)
             {
                 Log.Error($"Unable to start Garbage Collection Event Listener Sample.  Further .NetCore GC metrics will not be captured.  Error: {ex}");
                 Stop();
+                Dispose();
```

Comment:
is there a reason why we call Stop() and Dispose() ? Can we just call Dispose () ?

Reply:
no good reason, there's a bad reason, but I won't bore you with that :)